Frameworks/UIKit/UIView.mm

```diff
@@ -198,24 +199,68 @@ - (bool)_processGesturesForTouch:(UITouch*)touch event:(UIEvent*)event touchEven
         }
     }
 
-    // gesture priority list
-    const static id s_gesturesPriority[] = {[UIPinchGestureRecognizer class],
-                                            [UISwipeGestureRecognizer class],
-                                            [UIPanGestureRecognizer class],
-                                            [UILongPressGestureRecognizer class],
-                                            [UITapGestureRecognizer class] };
```

Comment:
Removing this probably breaks gesture chaining for UIPanGestureRecognizer and UITapGestureRecognizer. That is to say, you haven't implemented the delegates on the system gesture recognizers.